mLRS/Common/setup.h

```diff
@@ -398,6 +406,20 @@ void setup_configure(void)
         break;
     default:
         while (1) {} // must not happen, should have been resolved in setup_sanitize()
+    case SETUP_FREQUENCY_BAND_433_MHZ:
```

Comment:
not so nice, this all should come before the final default:
pl swap

Reply:
Updated.